include/xtensor/xfunction.hpp

```diff
@@ -39,6 +39,12 @@ namespace xt
 
     namespace detail
     {
+        // need to have a copy of meta_identity here for MSVC
+        template <class T>
+        struct meta_identity
+        {
+            using type = T;
+        };
```

Comment:
Is it still required? I think we can keep the one in `xutils.hpp` and remove this one now that `get_buffer_storage` is fixed.